internal/backuprestore/local_restore.go

```
@@ -0,0 +1,332 @@
+			return true, nil
+		}
+		if isTransientError(err) {
+			return false, nil
+		}
+		return false, err
+	})
+}
+
+func validateChecksums(backupPath string) error {
+	checksumFile := filepath.Join(backupPath, "checksums.sha256")
+	data, err := os.ReadFile(checksumFile)
```

golangci-lint (2.12.2)

[medium] G304: Potential file inclusion via variable

(gosec)